instance/models/openedx_instance.py

```diff
@@ -415,12 +410,18 @@ def terminate_obsolete_appservers(self, days=2):
     def shut_down(self):
         """
         Shut down this instance.
-
-        This process consists of two steps:
-
-        1) Disable New Relic monitors.
-        2) Terminate all app servers belonging to this instance.
         """
-        self.disable_monitoring()
```

Comment:
@smarnach Based on just this one commit it looks like this is getting removed completely? I think `shut_down` should call `disable_monitoring` (before terminating any app servers).